stumpy/motifs.py

```diff
@@ -0,0 +1,313 @@
+# STUMPY
+# Copyright 2019 TD Ameritrade. Released under the terms of the 3-Clause BSD license.
+# STUMPY is a trademark of TD Ameritrade IP Company, Inc. All rights reserved.
+
+import logging
+
+import numpy as np
+
+from . import core
+
+logger = logging.getLogger(__name__)
+
+
+def _fill_result_with_nans(top_Q_indices, top_Q_values):
+    max_number_of_occurrences = 0
```

Comment:
You may have missed it in my [previous reply](https://github.com/TDAmeritrade/stumpy/pull/185#issuecomment-795862345) (I edited it after the fact) but:

> I like that the jagged list is not wasteful in terms of space but it also doesn't benefit from faster operations, say, for computing summary statistics. I wonder if we could make use scipy.sparse instead? So, we start with your proposal of using a list-of-lists and then convert it to a scipy.sparse array before returning it to the user? This way, we get the best of both worlds of a compact representation of information and fast math operations on the sparse matrix?

I don't know...

Reply:
Oh I did not see this, indeed. To be honest, I don't think this is a big issue. I like having it a numpy array instead of a list, to be consistent across the whole package. Spacewise, it might be a bit wasteful. However, Usually the user will look at most at the top 20 motifs and be interested in the top 1000 occurrences (a very conservative estimate) and a 20x1000 numpy array is still really small.

Maybe we should keep it like this for now, until somebody experiences trouble.

Reply:
Perfect! I agree with everything that you said. Let's keep it as `ndarray`